great-idea/index.html

```diff
@@ -14,10 +14,98 @@
   <![endif]-->
 </head>
 
-  <body>
+  <body>  
+    <div class = "container">
+      <header>
+    <nav>
+  <a href ="file:///C:/Users/e0149196/Desktop/Personal%20Info/Lambda%20fullstack/Git/UI-III-Flexbox-master/UI-III-Flexbox-master/great-idea/services.html">Services</a>
+  <a href ="file:///C:/Users/e0149196/Desktop/Personal%20Info/Lambda%20fullstack/Git/UI-III-Flexbox-master/UI-III-Flexbox-master/great-idea/product.html">Product</a>
```

Comment:
I'd recommend providing relative paths here to your files. Since they're in the same directory, you could simplify these hrefs by simply providing `./services.html` and `./product.html`. You could even leave out the `./` if you wanted and it'd have the same effect.